k8s/sandbox/cluster-00/jenkins-test/jenkins-pvc.yaml

```diff
@@ -0,0 +1,32 @@
+
+apiVersion: "v1"
+kind: "PersistentVolume"
+metadata:
+  name: "jenkinspv"
+spec:
+  capacity:
+    storage: "20Gi"
```

Comment:
can you make this larger (500 GB)

Reply:
sorry didn't realise this was just a tests - for mgmt-sbox it will need to be larger :-)